substrate/primitives/runtime/src/traits/transaction_extension/as_transaction_extension.rs

```diff
@@ -1,130 +0,0 @@
-// This file is part of Substrate.
```

Comment:
Can't remove this - it's also for other people!

Reply:
Didn't think of that, I reverted the removal.